integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImage.java

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.logging.Level;
+
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.LoggerHelper;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.Operator.RestCertType;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Simple JUnit test file used for testing Model in Image.
+ *
+ * <p>This test is used for creating domain using model in image.
+ */
+
+public class ItModelInImage extends MiiBaseTest {
+  private static Operator operator;
+  private static String domainNS;
+  private static String testClassName;
+  private static StringBuffer namespaceList;
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception exception
+   */
+  @BeforeAll
+  public static void staticPrepare() throws Exception {
+    namespaceList = new StringBuffer();
+    testClassName = new Object() {
+    }.getClass().getEnclosingClass().getSimpleName();
+    // initialize test properties and create the directories
+    initialize(APP_PROPS_FILE, testClassName);
+  }
+
+  /**
+   * This method gets called before every test. It creates the result/pv root directories
+   * for the test. Creates the operator and domain if its not running.
+   *
+   * @throws Exception exception if result/pv/operator/domain creation fails
+   */
+  @BeforeEach
+  public void prepare() throws Exception {
+
+    createResultAndPvDirs(testClassName);
```

Comment:
Do we have PV directories for model-in-image tests?

Reply:
@anpanigr @vanajamukkara This question is for the QA team.

Reply:
logs are written on PV